UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Models/TableEntities/BessProductVersionEntities.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Azure;
+using Azure.Data.Tables;
+
+namespace UKHO.PeriodicOutputService.Common.Models.TableEntities
+{
+    [ExcludeFromCodeCoverage]
+    public class BessProductVersionEntities : ITableEntity
```

Comment:
Can we use existing ProductVersionEntities.cs class for it and if yes we can delete this as its almost same ?

Reply:
Resolved